docs/releasing.md

```diff
@@ -1,116 +1,141 @@
-   GitHub workflow should have automatically created draft releases
-   for you. Please make sure the release notes are checked
-   thoroughly before they are published.
+- For any directory with `go.mod` in it (excluding `hack/tools`), create another
+  Git tag with directory prefix, ie. `git tag -s -a api/v1.x.y -m api/v1.x.y`.
+  In this repository, those directories are `api` and `tests`.
+  This is enables the tag to be used as a Go module version for any downstream
+  users.
 
-#### Manual Process
+### Release notes
 
-1. Run `make release-notes` to create the release notes . Copy the output and sort
-   manually the items that need to be sorted.
-1. Create annotated tags:
+Next step is to clean up the release note manually.
 
-   ```bash
-   git tag -a -s v0.x.y -m v0.x.y
-   git tag -a -s test/v0.x.y -m test/v0.x.y
-   ```
+- Check for duplicates, reverts, and incorrect classifications of PRs, and
+  whatever release creation tagged to be manually checked.
+- For any superseded PRs (like CAPI uplifts, or commit revertions) that provide
+  no value to the release, create a summary line in `Superseded` section with
+  the PR ids and summary title. This way thye are acknowledged to be part of
+  the release, but not overwhelming the actual changes contained by the release.
+- Save the draft.
 
-   To use your GPG signature when pushing the tag, use `git tag -s [...]` instead
-1. Push the tags to the GitHub repository:
+### Tagging the Ironic and Mariadb images
 
-   ```bash
-   git push origin v0.x.y
-   git push origin test/v0.x.y
-   ```
+NOTE: This part will be moved to BMO releasing after BMO release process and
+maintenance plan has been agreed on.
 
-   NB: `origin` should be the name of the remote pointing to
-   `github.com/metal3-io/cluster-api-provider-metal3`
-1. Run `make release` to build artifacts (the image is automatically built by CI)
-1. [Create a release in GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
-   that contains the elements listed above that have been created in the `out`
-   folder
-1. Create a branch `release-0.x` for a minor release for backports and bug fixes.
-   This is not needed for patch releases.
+Before publishing CAPM3 release, we need to tag corresponding
+[Ironic-image](https://github.com/metal3-io/ironic-image) and
+[Maridb-image](https://github.com/metal3-io/mariadb-image) repositories.
+Those repositories do not have separate releasing process, but for the sake of
+backwards compatibility, each of the CAPM3 releases is tagged in these
+repositories as well.
 
-### Permissions
+```bash
+# Ironic
+git clone git@github.com:metal3-io/ironic-image.git
+cd ironic-image
+# if retagging old tag, add capm3-v1.x.y^{} to the line, e.g.
+# git tag -s -a capm3-v1.x.y -m capm3-v1.x.y capm3-v1.x.y^{}
+# this reuses the tag and resolves it into a sha for tagging
+git tag -s -a capm3-v1.x.y -m capm3-v1.x.y
+git push origin capm3-v1.x.y
```

Comment:
I do not have an exact suggestion, but could we reuse this block later instead of repeating it and somehow say: `follow the exact same steps as above` ?

Reply:
I had it like that before, I'll change it back.